packages/connect/src/ScProvider.ts

```diff
@@ -90,8 +90,11 @@ export class ScProvider implements ProviderInterface {
   ) {
     const isExtension = !!document.getElementById("substrateExtension")
 
+    const id = parseInt((Math.random() * 1000000000).toString(), 0)
```

Comment:
Im not really fun of this Math.random... If anything better come to mind please share @josepot